docs/reference/modules/shard-ops.asciidoc

```diff
@@ -0,0 +1,69 @@
+[[shard-allocation-relocation-recovery]]
+=== Shard allocation, relocation, and recovery
+
+Each <<documents-indices,index>> in Elasticsearch is divided into one or more <<scalability,shards>>, each of which acts as a copy of the index and can be replicated across multiple <<modules-node,nodes>> to protect against hardware failures.
```

Comment:
This sentence is confusing two different concepts, namely "shard" and "shard copy". An index is divided into ≥1 _shards_, each of which has ≥1 _shard copies_. Replication happens across the shard copies which together make up a single shard. Thus a shard copy holds the same set of documents as every other copy of that same shard, but distinct shards hold mutually disjoint sets of documents.

Each shard has one distinguished shard copy called the primary, and zero or more other shard copies called replicas (not sure if that's relevant here).

The docs are a little sloppy with this terminology in various places, saying "shard" when we mean "shard copy", and normally it's clear from context what we mean, but in cases like this it's probably helpful to be more precise.